### ISSUE TITLE
Mediation and usage of shared resources

### ISSUE THREAD

**Issue by orthecreedence** [basisproject/tracker#86]
We're going to start with three ideas:

- Ownership
- Stewardship
- Use

### Ownership

In the Basis, the network owns all resources. Members of the network are also owners of the network, therefor all network resources are owned by the members: resources are all held in common. This goes for property, inputs to production, capital, etc etc.

### Stewardship

Stewardship is granted to a company under the premise that the value of that resource, both in its use and the decisions surrounding it, is maximized by that company managing it. In other words, it probably makes sense for a house to be under stewardship of a housing company that operates in the area the house is located. It might not make sense for decisions about the management of the house or its use to be decided globally, or by a company 500 miles away from the house. There is no algorithm for determining optimal stewardship, but it's rather a decision between companies and people within the system.

Stewardship might have multiple entry points. For instance, a productive company that orders a widget to use in their productive processes would be automatically in stewardship of the widget. If a housing company purchases an apartment building out of the market using a capital pool (#92) then that housing company would automatically be in stewardship of the house.

For other things, stewardship starts at the global level, and it passed down to companies from there based on governance processes. In other words, if there's no pre-determined entry point for a resource, then it defaults to being in the global scope, managed by everyone and available for use by everyone.

### Use

Use is when someone is actually *using* a resource. A widget might get used in the productive process to make a chair. A tractor might be used to aid in farming. A house might be used to live in. Etc.

Use is determined by the steward of a resource. A regional housing company that is in stewardship of houses might decide that nobody can use two houses at once unless all members are housed (and there are leftover houses afterwards). A farming company might decide that anyone using the farm land they steward must care for the land in specific ways.

Use is somewhat special because it mirrors the idea of "ownership" in a market system. The idea being, when you're using a resource, it's more or less "yours" provided you are acting within the bounds set by the stewards. It's the idea of "yes, the house is yours, but no you cannot burn it to the ground just because you feel like it." Use also implies either shared or full responsibility of costs: if you're using a house or factory, you need to maintain it. If you're using a tractor, you might need to shoulder the cost of repairs. Stewards might pass through costs directly, or might assign costs using a time-based approach (what might be rent in a market system) with the caveat that only costs assigned to that resource may be assigned to the one using it (ie, a steward cannot *profit* from renting out a resource to a user, they can only assign existing costs).

## Goals

The idea here is that everyone owns everything, the people who are *most likely* to use some resource or be affected by it are stewards of that resource, and within that group members are free to use the resource within the guidelines set by themselves (the stewards).

This creates a system of *managed* commons where ownership is shared and use is open to anyone.

## Implementation

This is copied pretty much directly from #59 so it could be closed/merged into this one issue.

Asset management probably makes sense to set up as a public company, so it can take orders from other companies and purchase/sell assets from/to non-members based on regional need.

Management of assets would be based on a set of permissions:

- [ ] Permission for creating orders for acquiring assets
- [ ] Permission for putting assets up for sale on the public market
- [ ] Permission for arbiting use of assets
- [ ] Permission system should be tag-aware: a user might be able to manage housing assets but not farm land

Abilities:

- [ ] Add/remove available assets to/from system
- [ ] Ability to tag assets (housing, farm land, factories, etc) to assist in categorization but also permissions
- [ ] Ability for a company to "reserve" an asset (mark it in-use)
  - [ ] Usage can have a term (end date)
- [ ] Costing
  - [ ] Maintenance costs are taken on by those in-use of asset
  - [ ] Market rate costing for renting asset to non-members (set by company in use of asset, possibly with some restrictions)
  - [ ] Per-member costing based on membership percentage (member pays at-cost + difference between at-cost and market-rate depending on their status)

## Ideas/thoughts/notes:

- Stewardship can be assigned downward to smaller companies.
- How is stewardship assigned back upwards? This needs some real thought, because we don't want one-person companies controlling large amounts of property, and there needs to be a process by which the system handles this.
  - Hypothetical: if a housing company somehow has only one owner. Can the San Francisco city company assign stewardship to themselves to effectively eliminate what might be a landlord situation?
  - Now, if the housing company is not a member of the san francisco company, can they still do this?
  - A company that has no ties with another company might need some way of voting to appropriate either underused or overprovisioned resources in general.
  - Solutions:
    - Force all companies that own real property to be a member of a regional company (aka, everyone back to the regional socialism idea...)
      - Requires defining and moderating what IS real property. Seems like a lot of work to satisfy one hypothetical.
    - Incentives: you only get UBI if your company is a member of a geographical hierarchy (aka, the San Francisco company et al).
      - Ok, so I join the "earth" company and keep my 100 houses because good luck getting everyone on the planet to vote on this fringe issue
    - Allow people to appropriate property by vote
      - Somewhat chaotic. Can someone in Canada vote for my house to be under their stewardship? Can't trust Canadians...
    - **WINNER** Associate property and people geographically implicitely. This is kind of like forcing companies to be members of a regional company, but a) not as brittle and b) more closely mirrors how things work in the real world.
      - Perhaps if you live or work within N miles of a property, you have some say in its management, depending on your distance from it.
      - Taking this further, it's less about your home or work address, and more about where you spend your time. If there were a privacy-friendly way of using this data to determine your level of involvement over some property, then it woudl be ideal. I think until something like that materializes, just using an address makes sense for now.
      - This is getting into the territory of "people decide things based on how much it affects them" but uses geographical proximity as a defined and well-understood way of measuring affect. It's not perfect, but it's what we've got, and best of all: people already understand it.
- There needs to be a way to pass costs automatically from each resource to the agent using it. This would likely be time-based, because time represents an opportunity cost to others who might want to use that resource. Basically, we're charging rent (keep in mind, this is only until the cost of the asset has been transferred back into the economy, at which point the asset will be "free" to use).
- Time limits on usage? It might make sense for some resources to have a defined begin/end period, after which the resource becomes available for others to use again.
- There needs to be some codified way of determining use, almost like a community scripting language. __Just realized I reinvented smart contracts__. The goal here is to define a transparent set of rules, decided on by the community, that can be applied in an automated fashion based on past and present data. What we're doing here is replacing market allocation of resources with direct social intent.
  - Request use via order (intent/commitment)
    - First come first serve?
    - Lottery?
    - Vote?
  - Revoke use
  - Ability to tag resources? This ties in with #74 
  - Classifying resources by tag and matching usages with tags (ie, a house tagged "4br" might only be available to a family of 4+)?
  - Do you already have use of a house? Maybe you don't get another one.
  - Prioritize based on company performance metrics (#78)?

**Comment by orthecreedence** [comment]
So this would be some mix of smart contract (likely a predefined library of them with the option for custom contracts) but also mix in normal legal contract, ie "your use of this forest is predicated on only consuming X trees per day or up to Y% overall"

**Comment by orthecreedence** [comment]
Here is an interesting discussion on "property" in anarchism: https://www.reddit.com/r/DebateAnarchism/comments/hr63tx/property_use_and_determination_of_use/

Also, [this reddit post](https://www.reddit.com/r/basisproject/comments/hfdd4d/ubi_79_issues_basis_tracker/) has some great discussion on mathematics and influence over resources.

I don't want ideas here to get too esoteric that the layperson cannot understand them, but I do want to not just end up reinventing capitalist property norms in the first real implementation of the project.

There has to be some middle ground, and property relations can grow from the first version. After all, none of this is set in stone, even after the code is written...

**Comment by orthecreedence** [comment]
It's important to note that the current model has a somewhat capitalist notion of property: companies are in ultimate control of various property. In socialism, the factory might be socialized and the factory workers "in use" of it. Under anarchist property relations, the factory might be managed by a large number of people influenced by its use, need, operations, outputs, geographical proximity, and ecology. Quantifying this is interesting. Geography is an easy place to start, but not comprehensive.

Can the impact of a various decision about a resource even be quantified in a meaningful way that's not personal? This makes me wonder if giving people some limited number of votes over a period of time along with quadratic voting would be sufficient. The more an issue affects me, the more I might vote (using up my limited voting power which limits my decisions on future votes). This way even someone across the globe could vote on something local, but they might not because there is an economic factor involved. That said, it would be possible for large numbers of people to collude to vote a single way (only using one vote) to enact a change that might not affect them a lot but affects others a large amount. Why would they do this? Not sure.

So the regional model was effectively a crystalized socialist model: companies are members of a bigger group, and the bigger group owns the land. In the new company model (#72) it's not obvious what's to stop a single company from owning 17 farms. There needs to be some mechanism by which a company that is not a member of other (larger) companies can have their possession of property contested or altered by other members in some way.

In other words, there's the global company, and all its trees of sub-companies. We don't want all property to sit in the leaves of those trees, because then we just have capitalism. This is why I'm starting to lean towards anarchist models of property management now that I'm leaving the regional model. Lots to think about...

**Comment by orthecreedence** [comment]
What I'm playing with in my head is something like this: there's a global systemic company to start out. Just one. From there, people can create sub-companies, and take resources from the company they are forming out of. Thing is, the split has to be ratified by the original company.

This has the interesting property that *everything* is *everyone's* to begin with, and subdivides out from there, based on consent of existing region. This promotes default shared ownership, while allowing to drill down from there, as opposed to starting from the bottom (ie, a company owning a tractor) and building up from there (why build up at all? why not retain your resources?).

**Comment by orthecreedence** [comment]
This has been sitting in my head for so long now, and I think it needs a conclusion. There's no real consensus on how property might be managed in a post-capitalist system (unless you centralize it under a bureaucratic state, fun...) so for now this just needs to be open-ended, and I am going to put it under the pervue of company management and update the one unanswered question above with a poorly thought out solution.

**Comment by orthecreedence** [comment]
This issue is now effectively defined and in the paper, so I'm closing and kicking out the remaining details to other issues (linked above).
